File: API/ContainerNinja.Contracts/DTO/CalledIngredientDTO.cs
using ContainerNinja.Contracts.Data.Entities;

namespace ContainerNinja.Contracts.DTO
{
    public class CalledIngredientDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public KitchenProductDTO KitchenProduct { get; set; }
        public float? Amount { get; set; }
        public int KitchenUnitType { get; set; }
    }
}

File: Client/src/app/models/CalledIngredientDTO.ts
import { KitchenProductDTO } from "./KitchenProductDTO";

export class CalledIngredientDTO {
  id?: number = undefined;
  name?: string = undefined;
  kitchenProduct?: KitchenProductDTO = undefined;
  amount?: number | undefined = undefined;
  kitchenUnitType?: number = undefined;
  kitchenProductId?: number = undefined;
}
